### PR TITLE
refactor(es_extended,esx_notify,skinchanger): Updated prints

### PR DESCRIPTION
**Pull request by thesysadmindev**
Updated prints to use formatting on strings instead of concatenation.

### CODE REVIEW

File: [core]/es_extended/server/commands.lua [hunk 1]
```diff
@@ -347,25 +347,26 @@ ESX.RegisterCommand('saveall', 'admin', function()
...
-ESX.RegisterCommand('group', {"user", "admin"}, function(xPlayer)
-	print(xPlayer.getName()..", You are currently: ^5".. xPlayer.getGroup() .. "^0")
+ESX.RegisterCommand('group', {"user", "admin"}, function(xPlayer, _, _)
+	print(('%s, you are currently: ^5%s^0'):format(xPlayer.getName(), xPlayer.getGroup()))
 end, true)
 
-ESX.RegisterCommand('job', {"user", "admin"}, function(xPlayer)
-	print(xPlayer.getName()..", You are currently: ^5".. xPlayer.getJob().name.. "^0 - ^5".. xPlayer.getJob().grade_label .. "^0")
+ESX.RegisterCommand('job', {"user", "admin"}, function(xPlayer, _, _)
+	print(('%s, your job is: ^5%s^0 - ^5%s^0'):format(xPlayer.getJob().name, xPlayer.getJob().grade_label))
```
Comment (TheFantomas) [inline [core]/es_extended/server/commands.lua:355]: Forgot xPlayer.getName() to format() argument

First version:
```lua
ESX.RegisterCommand('group', {"user", "admin"}, function(xPlayer)
	print(xPlayer.getName()..", You are currently: ^5".. xPlayer.getGroup() .. "^0") -- xPlayer.getName(
end, true)
```
Your version:
```lua
ESX.RegisterCommand('group', {"user", "admin"}, function(xPlayer)
	print(('%s, your job is: ^5%s^0 - ^5%s^0'):format(xPlayer.getJob().name, xPlayer.getJob().grade_label))
end, true)
```

Fixed version:
```lua
ESX.RegisterCommand('group', {"user", "admin"}, function(xPlayer)
	print(('%s, your job is: ^5%s^0 - ^5%s^0'):format(xPlayer.getName(), xPlayer.getJob().name, xPlayer.getJob().grade_label))
end, true)
```

**Review by TheFantomas — CHANGES_REQUESTED** [review]

**Comment by Thekuca** [comment]
also fix conflicts

File: [core]/es_extended/server/commands.lua (hunk 1 above)
Comment (thesysadmindev) [reply [core]/es_extended/server/commands.lua]: Fixed!

**Review by Gellipapa — APPROVED** [review]
Good Job.

**Pull request merged**